watertap/unit_models/tests/test_anaerobic_digestor.py

```diff
@@ -407,7 +407,7 @@ def test_costing(self, adm):
         assert_optimal_termination(results)
 
         # Check solutions
-        assert pytest.approx(1083290.8, rel=1e-5) == value(
+        assert pytest.approx(2.0 * 1083290.8, rel=1e-5) == value(
```

Comment:
How does doubling the capital costs via this factor not affect LCOW? I don't see any LCOW tests changed.

Reply:
The factor is now part of the unit model's `capital_cost` variable, as opposed to being applied at the global level.

Reply:
Ok I see. 

Explanation for myself and anyone else: ``factor_capital_cost`` was set to 1 so mooted the influence of having ``TIC`` applied at the unit model level. So the value of ``aggregate_capital_cost``, which is the sum of unit model ``capital_cost``,  didn't change; ``aggregate_capital_cost``  is used in ``total_capital_cost``, which is used in ``LCOW``.

Reply:
For others, this change is being made to create uniformity around "where" the indirect capital cost is applied.

Right now the ZOCostingPackage does things at the unit-level, and the WaterTAPCosting package at the global-level. The PR modifies all the existing detailed models to put it at the unit-level.